src/app/portfolio/_components/ProjectCard.tsx

```
@@ -0,0 +1,86 @@
+  isOwner?: boolean;
+}
+
+export default function ProjectCard(props: ProjectCardProps) {
+  return (
+    <article className="flex flex-col gap-4 rounded-lg border-2 border-gray-300 bg-gray-100 p-5 text-text-primary">
+      <div>
+        <div className="flex items-center pb-2">
+          <h2 className="grow text-3xl font-semibold">{props.title}</h2>
+          {props.isOwner && (
+            <div className="flex items-center gap-1">
+              <button className="p-1 font-semibold text-brand-tertiary">수정</button>
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)